genesis-image/build.rs

```diff
@@ -19,6 +19,8 @@ fn main() {
     save(&vm, &genesis_config, &storage_trie);
 }
 
+// Safety: genesis-image is only used in tests
+#[allow(clippy::unwrap_used)]
```

Comment:
On a second note, isn't this lint allowed by default? So we should explicitly deny it in `clippy.toml`. Also apparently there's a separate [setting](https://doc.rust-lang.org/clippy/lint_configuration.html#allow-unwrap-in-tests) for allowing it in tests automatically.

Reply:
Yes, this lint is allowed by default. I was running clippy with `-Dclippy::unwrap_used` CLI flag to find the `unwraps` to audit. I haven't added a CI job to check for unwraps, but we could do that now.